docs/marketplace/dev/feature-integration-guides/202200.0/marketplace-product-search-widget-feature-integration.md

```diff
@@ -0,0 +1,401 @@
+---
+title: Marketplace Product Search Widget feature integration
```

Comment:
we don't have such a Spryker feature. Please check and update existing IGs according to changes introduced in the epic.